jsf-showcase-webapp/src/test/java/com/liferay/faces/test/showcase/multimediaimage/ImageGeneralTester.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2000-2016 Liferay, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.liferay.faces.test.showcase.multimediaimage;
+
+import org.junit.Test;
+
+import com.liferay.faces.test.selenium.Browser;
+import com.liferay.faces.test.selenium.assertion.SeleniumAssert;
+import com.liferay.faces.test.showcase.output.OutputTester;
+
+
+/**
+ * @author  Kyle Stiemann
+ * @author  Philip White
+ */
+public class ImageGeneralTester extends OutputTester {
+
+	@Test
+	public void runImageGeneralTest() throws Exception {
+
+		Browser browser = Browser.getInstance();
+		navigateToUseCase(browser, "graphicimage", "general");
```

Comment:
The switch between `"image"` and `"graphicImage"` should be automated. I think the best way to do this would be to check if component prefix is `"h"`. If it is `"h"` then the component name should be `"graphicImage"`, otherwise it should be `"image"`. An easy way to get the component prefix would be to make [`DEFAULT_COMPONENT_PREFIX` in **`TesterBase.java`**](https://github.com/liferay/liferay-faces-showcase/blob/master/jsf-showcase-webapp/src/test/java/com/liferay/faces/test/showcase/TesterBase.java#L60) `protected` (it is currently `private`), so that `ImageGeneralTester` can access it.